src/main/java/org/sonarlint/intellij/SonarLintIntelliJClient.kt

```diff
@@ -137,6 +142,57 @@ object SonarLintIntelliJClient : SonarLintRpcClientDelegate {
         suggestionsByConfigScopeId.forEach { (configScopeId, suggestions) -> suggestAutoBind(findProject(configScopeId), suggestions) }
     }
 
+    override fun suggestConnection(suggestionsByConfigScope: MutableMap<String, MutableList<ConnectionSuggestionDto>>?) {
+        if (suggestionsByConfigScope != null && suggestionsByConfigScope.size == 1) {
+            val dto = suggestionsByConfigScope.entries.stream().findFirst().get()
```

Comment:
Following the previous comment, this should be removed, but just for information, you can simplify this by `suggestionsByConfigScope.entries.first` to get the first entry, or even `suggestionsByConfigScope.first`